pyperformance/compile.py

```diff
@@ -801,6 +803,7 @@ def getint(section, key, default=None):
         conf.directory = getfile('compile', 'bench_dir')
         conf.lto = getboolean('compile', 'lto', True)
         conf.pgo = getboolean('compile', 'pgo', True)
+        conf.jit = getstr('compile', 'jit', 'no')
```

Comment:
Maybe default to the empty string, in case the upstream default gets changed at some point?

```suggestion
        conf.jit = getstr('compile', 'jit', '')
```